client/src/components/ProductCard.js

```diff
@@ -30,16 +30,25 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const ProductCard = ({item}) => {
+const ProductCard = ({item, listId,getListRelations}) => {
   const classes = useStyles();
+
+  const removeButtonClick = async () => {
+      const res = await fetch(`list-to-products/${listId}/${item.id}`,{
+      method:'DELETE'
+      })
+      console.log(await res.json())
+      getListRelations()
```

Comment:
So, while you can do something like this, I think it would be better to 1) return the updated data from the backend (probably not ideal in this case), or 2) just update the local copy of the data

Reply:
@Mila-ashtari, can you look how to does this

Reply:
yes